egs/librispeech/asr/simple_v1/train.py

```diff
@@ -25,7 +25,7 @@
 from snowfall.common import setup_logger
 from snowfall.models import AcousticModel
 from snowfall.models.tdnn_lstm import TdnnLstm1b
-from snowfall.training.graph import TrainingGraphCompiler
+from snowfall.training.ctc_graph import CtcTrainingGraphCompiler as TrainingGraphCompiler
```

Comment:
I think that “as” obfuscates the intent in code below (IMO it’s a good idea to have the specific topology type in the class name where it’s used)

Reply:
Fixed as you suggested.